src/controller/python/chip/yaml/runner.py

```diff
@@ -655,7 +655,7 @@ def decode(self, result: _ActionResult):
             # TODO Once yamltest and idl python packages are properly packaged as a single module
             # the type we are returning will be formalized. For now TestStep.post_process_response
             # expects this particular case to be sent as a string.
```

Comment:
This comment needs to be removed, right?  And does `TestStep.post_process_response` still expect anything here?

Reply:
Well, we discussed about having something exposed from the `matter_yamltests` package to represent default success and default error. Those are still a bit raw at the moment. 
That said, this is definitively not a string so I will update the comment. Thanks!